controller/common/common_types.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ObjectIds:
+    internal_id: str = ''
+    object_uid: str = ''

```

Comment:
```suggestion
    uid: str = ''

```
no need to use `object` twice

Reply:
done